### PR TITLE
Setup TravisCI and configured PostgreSQL

### PR DESCRIPTION
**Pull request by doshivatsal7**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+before_script:
+  - psql -c "CREATE DATABASE Skill_Finder;" -U postgres
+  - psql -c "CREATE USER root WITH CREATEUSER PASSWORD 'pass@123';" -U postgres
+  - psql -c "ALTER USER root CREATEDB;" -U postgres
+  - python3 manage.py makemigrations
+  - python3 manage.py migrate
+
+script:
+  - flake8
+  - python3 manage.py test
```
Comment (AvaisP) [inline .travis.yml]: Add new line

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,9 @@
+SECRET_KEY=nad7kycy!op3$^p@37=xk!mcwmmv5zw6@*e$r+#m5ood2)@i78
+DEBUG=on
+ENGINE=django.db.backends.postgresql
+DB_NAME=skill_finder
+DB_USER=postgres
+DB_PASS=pass@123
+DB_HOST=localhost
+DB_PORT=5432
+
```
Comment (AvaisP) [inline .env.sample]: Remove line

File: .env.sample [hunk 3]
```diff
@@ -0,0 +1,9 @@
+SECRET_KEY=nad7kycy!op3$^p@37=xk!mcwmmv5zw6@*e$r+#m5ood2)@i78
+DEBUG=on
+ENGINE=django.db.backends.postgresql
```
Comment (AvaisP) [inline .env.sample]: Do we need this as a env var?

File: .env.sample [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+DB_NAME=skill_finder
```
Comment (AvaisP) [inline .env.sample]: DB_NAME=<DB_NAME> 
Do similar changes for all the variables below this

File: .env.sample [hunk 5]
```diff
@@ -0,0 +1,9 @@
+SECRET_KEY=nad7kycy!op3$^p@37=xk!mcwmmv5zw6@*e$r+#m5ood2)@i78
+DEBUG=on
```
Comment (AvaisP) [inline .env.sample]: Needs to be either True or False right?

Comment (doshivatsal7) [reply .env.sample]: Works either ways, but I'll change it to True/False

File: .env.sample (hunk 4 above)
Comment (doshivatsal7) [reply .env.sample]: Done

File: .env.sample (hunk 3 above)
Comment (doshivatsal7) [reply .env.sample]: No but I thought of keeping also DB variables together

File: .env.sample (hunk 2 above)
Comment (doshivatsal7) [reply .env.sample]: Done

File: .travis.yml (hunk 1 above)
Comment (doshivatsal7) [reply .travis.yml]: Done

**Review by akksshah — APPROVED** [review]

**Pull request merged**